app/javascript/mastodon/locales/en.json

```diff
@@ -895,6 +898,7 @@
   "status.reply": "Reply",
   "status.replyAll": "Reply to thread",
   "status.report": "Report @{name}",
+  "status.revoke_quote": "Remove my quote from @{name}’s post",
```

Comment:
This should be "Remove my post from @{name}'s post"

Reply:
Changed